rskj-core/src/main/java/co/rsk/peg/utils/FederationUtils.java

```diff
@@ -0,0 +1,16 @@
+package co.rsk.peg.utils;
```

Comment:
I wouldn't put this into a `utils` package. It's not a good idea to group classes under `utils` or `exception` packages, classes that are related to each other should be together. Ideally we'd have a `federation` package with all federation related classes in it, but in the time being let's just keep it under co.rsk.peg package